modules/core/include/opencv2/core/types.hpp

```diff
@@ -163,10 +163,12 @@ template<typename _Tp> class Point_
     Point_();
     Point_(_Tp _x, _Tp _y);
     Point_(const Point_& pt);
+    Point_(Point_&& pt) noexcept;
     Point_(const Size_<_Tp>& sz);
     Point_(const Vec<_Tp, 2>& v);
 
     Point_& operator = (const Point_& pt);
+    Point_& operator = (Point_&& pt) noexcept;
```

Comment:
Do we need overload with `const Point_&&`? Are there any benefits for std containers from that?
Should we replace this one with const?
Same questions are for constructor.

Reply:
Although const rvalue reference exist, I've yet to see them in practical examples. I don't think containers take advantage of them.